baybe/scaler.py

```diff
@@ -100,6 +101,9 @@ def fit_transform(  # noqa: D102
     ) -> Tuple[Tensor, Tensor]:
         # See base class.
 
+        # Delay importing torch to speed up BayBE loading
```

Comment:
Although @AVHopp suggested this in the meeting, I vote for removing these comments because:
* They will be all over the place
* We already have quite many other delayed imports, and there we don't mention it either

Please speak up, Alex, if you disagree 🙃

Reply:
How about CONTRIBUTING.md?

Reply:
I also think the comments are obsolete and a *small* section on CONTRIBUTING mentioning that chemistry and torch dependencies are lazily (ie inside function bodies) imported to allow fast package loading in cases where that functionality is not needed

it actually means that all future contributions need to be checked to not break the lazy loading currently under development